src/mui-setup.js

```diff
@@ -1,11 +1,11 @@
 import { createGenerateClassName, createMuiTheme } from '@material-ui/core/styles';
 
-export default function muiSetup(isActive, sproutBase) {
+export default function muiSetup(sproutBase) {
   if (sproutBase.overrides) {
     /* eslint-disable no-param-reassign */
     sproutBase.overrides.MuiTableContainer.root.height = 'calc(100% - 52px)';
-    sproutBase.overrides.MuiTableContainer.root.overflow = !isActive ? 'auto' : 'hidden';
-    sproutBase.overrides.MuiPaper = { root: { height: '100%' } };
```

Comment:
This was applied to the rows per page dropdown as well.

Reply:
that one drove me crazy